src/NuGet.Indexing/StorageLoader.cs

```diff
@@ -61,5 +87,12 @@ public bool Reload(IndexingConfiguration config)
                 !(oldDataContainerName == _dataContainerName &&
                   oldStorageAccountConnectionString == _storageAccountConnectionString);
         }
+
+        private CloudBlockBlob GetBlob(string blobName)
+        {
+            var client = _storageAccount.CreateCloudBlobClient();
```

Comment:
Low priority, but we could rewrite `Reload` to save the client and then reuse it until `Reload` is called again.